### PR TITLE
chore: upgrade to compose v2

### PR DESCRIPTION
**Pull request by 0xc0000007b**
closes: #63

### CODE REVIEW

**Comment by 0xc0000007b** [comment]
@FantasticoFox @rht note the steps to upgrade your dev environments. if you cannot/don't want to upgrade your dev environment right now no problem as `docker-compose` is not yet deprecated. Here are the linux upgrade instructions (very simple): https://docs.docker.com/compose/cli-command/#install-on-linux

File: README.md [hunk 1]
```diff
@@ -2,16 +2,14 @@
 Author: inblockio / 23. August 2021
 
 Installation for Linux (Tested in Ubuntu 20.04) of your Personal Knowledge Container\
-This should work for other environments like Windows and Mac if the dependancies are fullfilled.
+This should work for other environments like Windows and Mac if the requirements are met.
```
Comment (rht) [inline README.md:5]: Shouldn't there be an extra empty line before the new subsection?

File: README.md [hunk 2]
```diff
@@ -20,49 +18,48 @@ If you want to install the PKC so that it is publicly accessible from the web:
...
 - Dockerized PKC https://github.com/inblockio/micro-PKC
-  For running the PKC MediaWiki including the DataAccounting extension via docker-compose.
+  For running the PKC MediaWiki including the DataAccounting extension via Docker Compose.
   languages: Shell, PHP, JavaScript (and Docker of course)
 - Content for population of a fresh installed pkc https://github.com/inblockio/PKC-Content
   This contains all the content which is pulled into the PKC through the create of an initial set of pages.
 - Smart Contract for Witnessing / Timestamping https://github.com/inblockio/DataSymmetry
   This Smart Contract is part of the infrastructure required to interact with the Ethereum network for time-stamping 
-  and 'proof of existance' of a set of verified page fiels.  
+  and 'proof of existence' of a set of verified page fields.
```
Comment (rht) [inline README.md:48]: Did you use `codespell` for linting the spellings? We should add `codespell` to CI, just like in the PKC-Content repo, i.e. https://github.com/inblockio/PKC-Content/blob/cf365475cb5c5f75f18e87fbd3fa867bf4586182/.github/workflows/ci.yml#L9-L17.

**Comment by FantasticoFox** [comment]
> @FantasticoFox @rht note the steps to upgrade your dev environments. if you cannot/don't want to upgrade your dev environment right now no problem as `docker-compose` is not yet deprecated. Here are the linux upgrade instructions (very simple): https://docs.docker.com/compose/cli-command/#install-on-linux

Already installed and ready to roll!

File: README.md (hunk 1 above)
Comment (0xc0000007b) [reply README.md:5]: I'd like to rewrite this entire README but it's not in scope for this PR. Let's plan later to make our README files across repos more consistent so we can tackle all README files at the same time.

File: README.md (hunk 2 above)
Comment (0xc0000007b) [reply README.md:48]: I've made a note and will take a look at that when I'm working on CI hardening. The downside to adding it to CI is that we'd then be in the business of maintaining an ever-growing dictionary, which would be more work than fixing misspellings in the editor while modifying files (using something like codespell, for example).

**Comment by 0xc0000007b** [comment]
closed by https://github.com/inblockio/micro-PKC/commit/82200f8367e563e5bf6b6446ab7a86ff2380e966

**Comment by 0xc0000007b** [comment]
Note GH typically closes related issues during rebase merges, however, this was manually closed which I'll investigate why that is in subsequent pulls.

**Comment by rht** [comment]
The dictionary won't be huge. E.g. this is the codespellignore for Zulip (48k of Markdown, 229k of Python): https://github.com/zulip/zulip/blob/main/.codespellignore.

**Comment by rht** [comment]
Maybe because you haven't force-pushed your latest version of compose-v2 to the GH remote, and so GH concludes that the commits are different.

**Pull request closed without merging**